common-docs/domains.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8">
+
+    <meta charset="UTF-8" />
+    <meta
+        name="Description"
+        content="This page is used to test the reachable web sites (domains) required for MakeCode functionality"
+    />
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0, user-scalable=yes"
+    />
+
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta
+        name="apple-mobile-web-app-status-bar-style"
+        content="translucent-black"
+    />
+    <meta name="format-detection" content="telephone=no" />
+    <meta
+        name="application-name"
+        content="Reachable MakeCode domains"
+    />
+
+    <meta name="msapplication-config" content="none" />
+    <meta name="theme-color" content="#3DC1AA" />
+
+    <title>Reachable MakeCode domains</title>
+
+    <link rel="stylesheet" href="https://pxt.azureedge.net/blob/2163189fd5e35c0981ed55318415582a7c9aeb12/doccdn/semantic.css" type="text/css">
+    <link rel="stylesheet" href="domains/style.css" type="text/css">
```

Comment:
typically we put css files under static e.g. `/static/domains/style.css`; in this case since it's such a small page already it might be better just to directly insert into a style tag here vs. dealing with backend fetching the right file / etc

Reply:
Good point. I think we need this page to be up irrespective of other pages. I will in line the style tag.